src/sorted_linked_list.rs

```diff
@@ -272,18 +272,56 @@ impl_index_and_const_new!(LinkedIndexU8, u8, new_u8, { u8::MAX as usize - 1 });
 impl_index_and_const_new!(LinkedIndexU16, u16, new_u16, { u16::MAX as usize - 1 });
 impl_index_and_const_new!(LinkedIndexUsize, usize, new_usize, { usize::MAX - 1 });
 
+mod as_view {
+    use super::{SortedLinkedList, SortedLinkedListIndex, SortedLinkedListView};
+
+    pub trait AsView<T, Idx, K>
```

Comment:
This is now a sealed trait visible in public docs. Could these methods be added to `SortedLinkedListSealedStorage` instead?

Reply:
Yeah I think they could but it would be confusing as you'd need the whole list and not just the storage, so both `as_view` and `as_view_mut` would take `&SortedLinkedListInner<T, Self>` instead of just `&self`

Reply:
Seems like https://github.com/rust-embedded/heapless/pull/500 does exactly that for all types.

Reply:
I'll rebase on top of that and see how to fix this then